### PR TITLE
feat(narrow-layout) Use drawer menus on desktop narrow mode

### PR DESCRIPTION
**Pull request by horymury**

### CODE REVIEW

File: react/features/participants-pane/components/breakout-rooms/components/web/RoomList.js [hunk 1]
```diff
@@ -55,14 +57,14 @@ export const RoomList = ({ searchString }: Props) => {
...
-                            onLeave = { lowerMenu }
+                            onLeave = { hideMenu }
                             onRaiseMenu = { onRaiseMenu(room) }
                             participantContextEntity = { raiseParticipantContext.entity }
                             raiseParticipantContextMenu = { raiseParticipantMenu }
                             room = { room }
                             searchString = { searchString }
                             toggleParticipantMenu = { toggleParticipantMenu }>
-                            {!_overflowDrawer && <>
+                            {!isMobileBrowser() && <>
```
Comment (saghul) [inline react/features/participants-pane/components/breakout-rooms/components/web/RoomList.js:67]: I guess you need to check for narrow mode here?

File: react/features/video-menu/components/web/LocalVideoMenuTriggerButton.tsx [hunk 2]
```diff
@@ -221,7 +221,7 @@ class LocalVideoMenuTriggerButton extends Component<IProps> {
                     overflowDrawer = { _overflowDrawer }
                     position = { _menuPosition }
                     visible = { popoverVisible }>
-                    {!_overflowDrawer && buttonVisible && !isMobileBrowser() && (
+                    {buttonVisible && !isMobileBrowser() && (
```
Comment (saghul) [inline react/features/video-menu/components/web/LocalVideoMenuTriggerButton.tsx:224]: doesn't this need the narrow check?

**Review by saghul — COMMENTED** [review]
Left some comments.

File: react/features/video-menu/components/web/LocalVideoMenuTriggerButton.tsx (hunk 2 above)
Comment (horymury) [reply react/features/video-menu/components/web/LocalVideoMenuTriggerButton.tsx:224]: for mobile the trigger for menu is long press; for desktop even when we have narrow(drawer menu) we still want the trigger for the menu to be the buttons.

File: react/features/participants-pane/components/breakout-rooms/components/web/RoomList.js (hunk 1 above)
Comment (horymury) [reply react/features/participants-pane/components/breakout-rooms/components/web/RoomList.js:67]: ditto :)

File: react/features/video-menu/components/web/LocalVideoMenuTriggerButton.tsx (hunk 2 above)
Comment (saghul) [reply react/features/video-menu/components/web/LocalVideoMenuTriggerButton.tsx:224]: Gotcha!

**Review by saghul — APPROVED** [review]
Let's go! 🚀

**Pull request merged**